llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -291,7 +291,7 @@ def _validate_config(self) -> None:
                     + 'pip install flash-attn==1.0.6 --no-build-isolation \n' +
                     'pip install git+https://github.com/NVIDIA/TransformerEngine.git@144e4888b2cdd60bd52e706d5b7a79cb9c1a7156'
                 )
-        if self.ffn_config['ffn_type'] in ['mptmlp', 'mptgeglu']:
```

Comment:
Can you add an error here if ffn type is mptgeglu that tells user the name changed and the structure changed slightly, linking to this pr for details. Just in case anyone was using it.

Reply:
[Done](https://github.com/mosaicml/llm-foundry/pull/829/commits/52be34f1d4e92dfae75b0e6dd1e947403c167e47)